src/utils/logger.ts

```diff
@@ -9,7 +10,7 @@ const Logger = {
    * @deprecated use `@/logger` instead, and see `@/logger/README` for documentation
    */
   debug(...args: any[]) {
-    if (__DEV__) {
+    if (__DEV__ && !device.get(['doNotTrack'])) {
```

Comment:
GREAT call adding to old logger too. I think some of these only emit in dev anyway though, so pry don't need this check on every method.